jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/AvatarTests.java

```diff
@@ -1,78 +1,44 @@
-        dataDisplayAvatarFrame.circle.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.circle.is().displayedImg();
-        dataDisplayAvatarFrame.circle.is().hasImg("MuiAvatar-img");
-    }
-
-    @Test
-    public void squareTest(){
-        dataDisplaySquareAvatarPage.open();
+import static io.github.com.pages.displaydata.AvatarPage.onlineStatus;
+import static io.github.com.pages.displaydata.AvatarPage.avatarWithPhoto;
+import static io.github.com.pages.displaydata.AvatarPage.avatarWithoutPhoto;
 
-        dataDisplayAvatarFrame.square.is().displayed();
-        dataDisplayAvatarFrame.square.is().hasClass("MuiAvatar-square");
-        dataDisplayAvatarFrame.square.is().displayedImg();
-        dataDisplayAvatarFrame.square.is().hasImg("MuiAvatar-img");
-    }
-
-    @Test
-    public void roundedTest(){
-        dataDisplayRoundedAvatarPage.open();
-
-        dataDisplayAvatarFrame.rounded.is().displayed();
-        dataDisplayAvatarFrame.rounded.hasClass("MuiAvatar-rounded");
-        dataDisplayAvatarFrame.rounded.is().displayedImg();
-        dataDisplayAvatarFrame.rounded.is().hasImg("MuiAvatar-img");
-    }
 
-    @Test
-    public void fallbackAltTest(){
-        dataDisplayFallbackAltAvatarPage.open();
 
-        dataDisplayAvatarFrame.fallbackAlt.is().displayed();
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("L");
+public class AvatarTests extends TestsInit {
+    @BeforeTest
+    public void beforeTest() {
+        //openSection("Avatar");
+        StaticSite.dataDisplayAvatarPage.open();
+        StaticSite.dataDisplayAvatarPage.isOpened();
     }
 
     @Test
-    public void letterTest(){
-        dataDisplayLetterAvatarPage.open();
-
-        dataDisplayAvatarFrame.letter.is().displayed();
-        dataDisplayAvatarFrame.letter.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("A");
+    public void avatarTests() {
+        testLogic(avatarWithoutPhoto);
+        testLogic(avatarWithPhoto);
+        onlineStatus.get(1).has().classValue(Matchers.containsString("MuiBadge-dot"));
+        onlineStatus.get(2).has().classValue(Matchers.containsString("MuiBadge-anchorOriginBottomRightCircle"));
     }
 
-    @Test
-    public void iconTest(){
-        dataDisplayIconAvatarPage.open();
-
-        dataDisplayAvatarFrame.icon.is().displayed();
-        dataDisplayAvatarFrame.icon.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.icon.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.icon.is().displayedSvg();
-        dataDisplayAvatarFrame.icon.is().hasSvg("MuiSvgIcon-root");
+    private void testLogic(List<UIElement> elements) {
+        for (int i = 1; i < elements.size(); i++) {
```

Comment:
Please check full list of elments

Reply:
It's already full list, it's just called two times because of two locators used for avatars.

Reply:
fixed upper border missing in list